rust/lang/security/audit/crypto/math-random.rs

```diff
@@ -0,0 +1,15 @@
+fn random() {
+    //ruleid: math-random-used
+    let x = rand::random();
+}
+fn thread_rng() {
+    //ruleid: math-random-used
+    let x = rand::thread_rng();
+}
+
+fn thread_rng_1() {
+    //todoruleid: math-random-used
+    use rand::thread_rng;
+    //todoruleid: math-random-used
+    let _ = thread_rng();
```

Comment:
Semgrep does some symbolic propagation and should be aware enough for this case to be captured with your current version of the rule:

https://semgrep.dev/playground/s/N0oy